ratelimits/limiter.go

```diff
@@ -277,10 +277,12 @@ func (l *Limiter) BatchSpend(ctx context.Context, txns []Transaction) (*Decision
 	batchDecision := allowedDecision
 	newBuckets := make(map[string]time.Time)
 	incrBuckets := make(map[string]increment)
+	expiredBuckets := make(map[string]time.Time)
```

Comment:
I think these names could be improved.

My bad for "incrBuckets", but that describes what we're going to do with those buckets, rather than describing a feature of the buckets themselves. Maybe "usedBuckets"?

And "expired" isn't super intuitive to me, since the bucket hasn't expired -- it's just full. So maybe "fullBuckets"?

Sorry for bikeshedding; naming is hard.